aliza/server.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const fs = require('fs');
+const http = require('http');
+const stream = require('stream');
+
+http.createServer((req, res) => {
+  if (req.url === '/notes' && req.method === 'GET'){
```

Comment:
Looks good - if you have some time we could talk about how to refactor to using returns instead of if/then/else.  I